app/assets/javascripts/oxalis/constants.js

```diff
@@ -91,6 +91,13 @@ export type ContourModeType = $Keys<typeof ContourModeEnum>;
 export const NODE_ID_REF_REGEX = /#([0-9]+)/g;
 export const POSITION_REF_REGEX = /#\(([0-9]+,[0-9]+,[0-9]+)\)/g;
 
+// The plane in orthogonal mode is a little smaller than the viewport
+// as there are two borders with width 2px each => 8px
+// TODO: Wouldn't 2px on each side only result in 4px additional width instead of 8px?
+export const OUTER_BORDER_ORTHO = 2;
```

Comment:
I'm seeing some movement to the outer edge by the scalebar when scaling with K/L. Maybe this will be fixed when changing this value to (the more correct?) 4, but it's just a guess.

Reply:
The actual border size also depends on the scale which would need to be incorporated. However, I managed to do this completely with percentages in css (since the border will always be approx. 2% of the input catcher).